doc/pattern_tools/gen_pattern.py

```diff
@@ -45,7 +50,7 @@ def make_circles_pattern(self):
         for x in range(0, self.cols):
             for y in range(0, self.rows):
                 dot = SVG("circle", cx=(x * spacing) + x_spacing + r,
-                          cy=(y * spacing) + y_spacing + r, r=r, fill="black", stroke="none")
+                          cy=(y * spacing) + y_spacing + r, r=r, fill="black", stroke="black", stroke_width = r*0.01)
```

Comment:
If I understand correctly from https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Fills_and_Strokes the stroke adds line around the object and it makes circles and squares larger. I do not think that it's what we want for calibration part of the pattern. For Aruco part it's ok to fill the gaps between pattern elements.